.travis.yml

```diff
@@ -1,8 +1,8 @@
 dist: focal
 language: node_js
 node_js:
+    - node
```

Comment:
As I checked, it is different from sandstone .travis.yml. Will the sandstone repo also be updated?

Reply:
That's my intention. Right. (If no issue occurs due to this change.)

Reply:
BTW, I want to change the order in this repo. I will update this.